packages/esm-service-queues-app/src/patient-queue-header/patient-queue-header.component.tsx

```diff
@@ -46,14 +47,20 @@ const PatientQueueHeader: React.FC<{ title?: string }> = ({ title }) => {
           </div>
           <div className={styles.dropdown}>
             <label className={styles.view}>{t('view', 'View')}:</label>
-            <Dropdown
-              id="typeOfCare"
-              label={currentQueueLocationName ?? queueLocations?.[0]?.name}
-              items={[{ display: `${t('all', 'All')}` }, ...queueLocations]}
-              itemToString={(item) => (item ? item.name : '')}
-              type="inline"
-              onChange={handleQueueLocationChange}
-            />
+            {isLoading ? (
+              <div style={{ width: 150 }}>
```

Comment:
1. Please do not use inline styles.
2. We usually use the styles in `rem` instead of `px`.